### DISCUSSION TITLE
Preserving Data Structure Between Source and Destination

### DISCUSSION THREAD

**Discussion by marcosmarxm** [airbytehq/airbyte#35447] [Connector Questions] [answered]
Hi. Is there a way to copy files AS IS from source to destination? Currently, when I try to copy a CSV file test.txt from source, that looks like:
```
Name,Age,Location
John,25,New York
Emma,30,Los Angeles
```
it becomes like this in the destination.
```
_airbyte_ab_id,_airbyte_emitted_at,_airbyte_data
"0141053...","1708...",{"Name":"John","Age":"25","Location":"New York"}
"ee0b078...","1708...",{"Name":"Emma","Age":"30","Location":"Los Angeles"}
```
 Is there a way to simply transport the file from source to destination without changing it? (or at least, not JSONify the contents of the original file)?
More info:
In my case, source and destination are both Azure Blob Storage.
For the source connector, I have tried both the [Files](https://docs.airbyte.com/integrations/sources/file) the [Azure Blob Storage](https://docs.airbyte.com/integrations/sources/azure-blob-storage) with file format set to CSV, but they both yield the same result as above.

**Comment by marcosmarxm** [comment, marked as answer]
Currently, preserving the original CSV structure when copying files with Airbyte is not supported. Airbyte converts records to an `AirbyteMessage(id, emitted_at, data)` format, which is essential for syncing different sources to various destinations. This standardization may not align with use cases that require maintaining the original data structure during migration.

**Reply by babrar** [reply]
I see. I feel that the need to sync files between cloud storages is quite common. Does [AirbyteHQ](https://github.com/airbytehq)  
 encounter such a requirement, and if it does, what tools or approaches do you employ to solve it?
